### PR TITLE
Add docker files

### PR DESCRIPTION
**Pull request by Levshunovma**

### CODE REVIEW

File: docker/micro-cluster-lab/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:bionic
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-amd64/jre
```
Comment (modelflat) [inline docker/micro-cluster-lab/Dockerfile]: usage of this syntax (without `=` sign) [is discouraged](https://docs.docker.com/engine/reference/builder/#env). I suggest that we use newer `ENV <key>=<value>` syntax across this file

File: docker/micro-cluster-lab/script_files/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+hdfs namenode -format
+service ssh start
+if [ "$HOSTNAME" = node-master ]; then
+    start-dfs.sh
+    start-yarn.sh
+    #start-master.sh
+    cd /root/lab
+    /opt/hadoop/bin/hadoop fs -put /spark_volume/spark_example.txt /spark_example.txt
+    /opt/spark/bin/spark-submit --class Example --master yarn /spark_volume/java-spark-1.0-SNAPSHOT.jar /spark_example.txt
```
Comment (modelflat) [inline docker/micro-cluster-lab/script_files/bootstrap.sh]: (question) do we really want to execute this sample code each time we start up our cluster?

File: docker/micro-cluster-lab/Dockerfile (hunk 1 above)
Comment (Levshunovma) [reply docker/micro-cluster-lab/Dockerfile]: Fixed

File: docker/micro-cluster-lab/script_files/bootstrap.sh (hunk 2 above)
Comment (Levshunovma) [reply docker/micro-cluster-lab/script_files/bootstrap.sh]: I think we don't, deleted it

**Review by modelflat — APPROVED** [review]
LGTM

**Review by Dmitriy-Ivanov-04 — APPROVED** [review]

**Pull request merged**